### PR TITLE
[Backport] Version bumps and `prdocs` reordering form 2412

### PR DESCRIPTION
**Pull request by EgorPopelyaev**
This PR includes backport of the regular version bumps and `prdocs` reordering from the `stable2412` branch back ro master

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1255,22 +1255,22 @@ dependencies = [
 
 [[package]]
 name = "assets-common"
-version = "0.18.0"
+version = "0.18.1"
```
Comment (gui1117) [inline Cargo.lock:1258]: To be exact 2412 doesn't actually use assets-common 0.18.1, it only uses 0.7.0.

File: Cargo.lock [hunk 2]
```diff
@@ -4464,18 +4464,18 @@ dependencies = [
 
 [[package]]
 name = "crossbeam-queue"
-version = "0.3.11"
+version = "0.3.12"
```
Comment (gui1117) [inline Cargo.lock:4467]: same stable2412 use 0.3.8

**Review by gui1117 — APPROVED** [review]
This PR also update minor version of various dependencies. But it seems fine.

File: Cargo.lock (hunk 1 above)
Comment (EgorPopelyaev) [reply Cargo.lock]: Yep, Cargo.lock should not be here, I reverted it back to the state how it was in master

**Review by lexnv — APPROVED** [review]

**Review by alvicsam — APPROVED** [review]

**Comment by EgorPopelyaev** [comment]
/cmd prdoc --audience runtime_dev --bump patch --verify false

**Pull request merged**